var/spack/repos/builtin/packages/llvm-amdgpu/package.py

```diff
@@ -94,6 +94,7 @@ class LlvmAmdgpu(CMakePackage):
         ),
     )
     variant("openmp", default=False, description="Enable OpenMP")
+    conflicts("+openmp", msg="llvm-amdgpu OpenMP support is currently disabled")
```

Comment:
This option is verified to not work with any version `@5:`.

Reply:
@estewart08 , need your help here.